drone/exec/exec.go

```diff
@@ -249,9 +253,14 @@ func exec(c *cli.Context) error {
 			c.StringSlice("network"),
 		),
 		transform.WithProxy(),
+		transform.WithEnviron(
+			readParams(
+				c.String("secret-file"),
```

Comment:
I guess that should be env-file

Reply:
Nope. 

Those are separate parameters, maybe the diff is not super clear:

```
		transform.WithEnviron(
			readParams(
				c.String("secret-file"),
			),
		),
		transform.WithSecrets(
			readParams(
				c.String("secret-file"),
			),
		),
```

and

``` 
		transform.WithEnviron(
			readParams(
				c.String("env-file"),
			),
		),
```

belong together. I think I had to add the WithEnviron(secret-file) because WithSecrets only hides it from the output, but doesn't add it to the env.

Reply:
Ack, just checked again. It needs WithEnviron and WithSecrets.

Reply:
at least otherwise `PASSWORD=********` does not show up in `env` :-)

Reply:
Ah, so the completely correct approach would be to inject only the requested secrets.. oh well :)

Dunno if that's not a bit overkill for my rust go skills :-D

Reply:
And back to the root of my comment, you define the env-file attribute, but you aren't using it :)